bin/codecept.js

```diff
@@ -95,7 +95,7 @@ program.command('run [test]')
   .option('--features', 'run only *.feature files and skip tests')
   .option('--tests', 'run only JS test files and skip features')
   .option('-p, --plugins <k=v,k2=v2,...>', 'enable plugins, comma-separated')
-
+  .option('--failed', 'to run failed/custom Tests')
```

Comment:
Please remove this option (as it affects core) and use environment variable instead:

```
RUN_FAILED=true npx codeceptjs run
```